src/patternfly/components/CodeEditor/examples/CodeEditor.md

```diff
@@ -0,0 +1,140 @@
+---
+id: Code editor
+section: components
+beta: true
+cssPrefix: pf-c-code-editor
+---
+
+## Examples
+### Default
+```hbs
+{{#> code-editor}}
+  {{#> code-editor-control}}
+    {{#> code-editor-action-group}}
+      {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Copy to clipboard"'}}
+        <i class="fas fa-copy" aria-hidden="true"></i>
+      {{/button}}
+      {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Download code"'}}
```

Comment:
Using an input group would get rid of the double border between buttons

<img width="400" alt="Screen Shot 2020-09-08 at 5 44 16 PM" src="https://user-images.githubusercontent.com/35148959/92534697-1505ac00-f1fb-11ea-89c2-fc8d4ae40743.png">